src/garage/experiment/experiment.py

```diff
@@ -258,11 +258,14 @@ def run_experiment(method_call=None,
         try:
             if env is None:
                 env = dict()
-            subprocess.call(command, shell=True, env=dict(os.environ, **env))
+            subprocess.run(command,
+                           shell=True,
+                           env=dict(os.environ, **env),
+                           check=True)
         except Exception as e:
+            import sys
```

Comment:
Why `import sys`?

Reply:
Oops! Missed deleting this when I was testing if calling sys.exit() here returns the error code in test.

Reply:
I'm surprised the precommit didn't catch this.